scrapy_po/requests_mixin.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from typing import Type
+
+from scrapy import Request
+from urllib.parse import urljoin
+
+
+class RequestsFromUrlsMixin():
+    """ Add a method `requests` that wraps urls returned by method `urls`
+    into ``Request``s and yield it so that can be used in a Scrapy callback
+    in the form of ``yield from obj.requests(...args...). Note that this
+    mixin expects the parent to have the ``urls`` and ``url`` methods defined """
+
+    def requests(self, *args, **kwargs):
+        for url in self.urls():
+            yield Request(urljoin(self.url, url),
```

Comment:
response.urljoin is better, because it handles base url properly

Reply:
Sorry, I should have thought about it more. I think switching to Scrapy 2.0 in examples is better than adding custom shortcuts to the codebase.